packages/agoric-cli/tools/getting-started.js

```diff
@@ -103,6 +103,7 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
     if (process.env.AGORIC_INIT_OPTIONS) {
       const opts = JSON.parse(process.env.AGORIC_INIT_OPTIONS);
       initOptions.push(...opts);
+      initOptions['--dapp-template'] = 'dapp-fungible-faucet';
     }
     t.is(
       await myMain(['init', ...initOptions, 'dapp-foo']),
```

Comment:
I think the easiest is to put the default options here

```suggestion
      await myMain(['init', '--dapp-template', 'dapp-fungible-faucet', '--dapp-base', 'https://github.com/Agoric/', ...initOptions, 'dapp-foo']),
```

Reply:
Ohhh I see. Updated the PR to do that and tests are passing now, thank you!

Omitted `--dapp-base` because it defaults to `"https://github.com/Agoric/"`

Ref:
```
  --dapp-base <base-url>  find the template relative to <base-url> (default: "https://github.com/Agoric/")
```

Reply:
Oh right `dapp-offer-up` moved to the `Agoric` org, so `DEFAULT_DAPP_URL_BASE` no longer changes in this PR!